src/bare-metal/aps/examples/exceptions.S

```diff
@@ -64,7 +65,9 @@
 	ldr x18, [sp, #8 * 18]
 	ldp x29, x30, [sp, #8 * 20]
 
-	/* Restore registers elr_el1 & spsr_el1, using x0 & x1 as scratch. */
+	/*
+     * Restore registers elr_el1 & spsr_el1, using x0 & x1 as scratch.
```

Comment:
this comment seems to have an incorrect indentation

Reply:
it looks like tabs vs. spaces

Reply:
Oh my, thanks for spotting this. I don't know why the comment was indented using a tab originally — I've now indented the other comments with tabs too.